Ports/vim/Vim.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+PORT_DIR=Vim
+INSTALLOPTS="DESTDIR=$SERENITY_ROOT/Root/"
+
+function fetch() {
+    run_fetch_git "https://github.com/vim/vim.git"
+}
+
+function configure() {
+    (
+        cd "$PORT_DIR"
+        cat > src/auto/config.cache << "DONE"
+        vim_cv_getcwd_broken=no
+        vim_cv_memmove_handles_overlap=yes
+        vim_cv_stat_ignores_slash=yes
+        vim_cv_tgetent=zero
+        vim_cv_terminfo=yes
+        vim_cv_toupper_broken=no
+        vim_cv_tty_group=world
+DONE
+
+        ./configure --host=i686-pc-serenity \
+                    --target=i686-pc-serenity --with-features=tiny \
+                    --with-tlib=ncurses
+    )
+}
+
```

Comment:
nit: I'd drop the newlines between functions since these files are going to be so tiny anyway. I want to try make them easier to "absorb" at a glance. But that's personal preference.